scripts/kms/key_rotation_policy_set.sh

```diff
@@ -14,9 +16,17 @@ key_rotation_policy_set() {
     # Echo the ROTATION_POLICY to the proposal file
     echo "$ROTATION_POLICY" | jq > "$WORKSPACE/proposals/set_key_rotation_policy.json"
 
-    # Submit the proposal
-    source $REPO_ROOT/scripts/kms/endpoints/proposals.sh \
-        "$WORKSPACE/proposals/set_key_rotation_policy.json"
+    (
+        # If running on sandbox_local, use the user cert because KMS can only
+        # authenticate user COSE signature
+        if [[ "$KMS_URL" == "https://127.0.0.1:8000" ]]; then
```

Comment:
Nit:Since you have to do this more than once, would some flag indicating that the KMS was a special local one or some such scheme be nicer?